src/Tuttifrutti/Json.hs

```diff
@@ -54,7 +54,7 @@ stripPrefix prefix =
     { Json.fieldLabelModifier
         = uncapitalize . fromMaybe (error ("Did not find prefix " ++ prefix)) . List.stripPrefix prefix
     }
-  where
-    uncapitalize :: String -> String
-    uncapitalize (head:rest) = Char.toLower head : rest
-    uncapitalize []          = []
+
+uncapitalize :: String -> String
```

Comment:
Add this to the export list as well https://github.com/KSF-Media/tuttifrutti/blob/master/src/Tuttifrutti/Json.hs#L2-L8